Sluggy.Tests/Strategies/NonAlphaNumericStrategyTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Sluggy.Strategies;
+using Xunit;
+
+namespace Sluggy.Tests
+{
+    public class NonAlphaNumericStrategyTests
+    {
+        [Trait("Project", "Sluggy")]
+        [Theory(DisplayName = "Should remove nonalphanumeric characters from string")]
+        [InlineData("<>*.,;´`'~^!#%$&/()=}{[]@£€§¨+|", "")]
+        [InlineData("a<>*.,;´`'~^!#%$&/()=}{[]@£€§¨+| -a", "a a")]
```

Comment:
Weird to have an example where the response has a space. Yes this is a correct behaviour but the strategy should never be computed with spaces (as it's split before)

Reply:
That makes sense, will fix the test.